### PR TITLE
Add new REV components

### LINKED ISSUE #1398: Add REV Components to Hardware Overview
Now that product details have been officially announced, obtain vectors and work with REV on adding necessary details. Opening this issue as a TODO for myself.

### PR DESCRIPTION
**Pull request by Daltz333**
Close #1398 

I opted to declare the manufacturer before each similar device to provide further clarification to the reader.

### CODE REVIEW

**Review by jasondaming — APPROVED** [review]

File: source/docs/hardware/sensors/serial-buses.rst [hunk 1]
```diff
@@ -106,6 +106,6 @@ Several sensors primarily use the CAN bus. Some examples include:
 - `CAN Based Time-of-Flight Range/Distance Sensor from playingwithfusion.com <https://www.playingwithfusion.com/productview.php?pdid=96&catid=1009>`__
 - TalonSRX-based sensors, such as the `Gadgeteer Pigeon IMU  <http://www.ctr-electronics.com/sensors/gadgeteer-imu-module-pigeon.html>`__ and the `SRX MAG Encoder <https://www.ctr-electronics.com/sensors/srx-magnetic-encoder.html>`__
 - `CANifier <https://www.ctr-electronics.com/control-system/can-can-canifier-driver-led-driver-gpio.html>`__
-- Power monitoring sensors built into the :ref:`Power Distribution Panel (PDP) <docs/controls-overviews/control-system-hardware:Power Distribution Panel>`
+- Power monitoring sensors built into the :ref:`CTRE Power Distribution Panel (PDP) <docs/controls-overviews/control-system-hardware:CTRE Power Distribution Panel>`
```
Comment (NoahAndrews) [inline source/docs/hardware/sensors/serial-buses.rst:109]: This item should also mention the PDH.

**Comment by jasondaming** [comment]
If we link the PDP there we should probably link the PDH too.

**Review by NoahAndrews — DISMISSED** [review]

File: source/docs/controls-overviews/control-system-hardware.rst [hunk 2]
```diff
@@ -25,23 +25,41 @@ NI roboRIO
...
-Voltage Regulator Module
-------------------------
+REV Power Distribution Hub
+--------------------------
+
+.. image:: images/control-system-hardware/power-distribution-hub.png
+  :alt: REV Power Distribution Hub
+  :width: 500
+
+The `REV Power Distribution Hub <https://docs.revrobotics.com/rev-11-1850/>`__ (PDH) is designed to distribute power from a 12VDC battery to various robot components. With 20 high-current (40A max) channels, 3 low-current (15A max), and 1 switchable low-current channel, the PDH gives teams more flexibility for overall power delivery. The Power Distribution Hub features toolless latching WAGO terminals, an LED voltage display, and the ability to connect over CAN or USB-C to the REV Hardware Client for real-time telemetry, making it easier than ever to wire and debug your robot.
```
Comment (sciencewhiz) [inline source/docs/controls-overviews/control-system-hardware.rst:44]: The part about flexibility reads more like marketing rather then documentation. I recommend removing.

File: source/docs/controls-overviews/control-system-hardware.rst [hunk 3]
```diff
@@ -25,23 +25,41 @@ NI roboRIO
...
+
+REV Radio Power Module
+----------------------
+
+.. image:: images/control-system-hardware/radio-power-module.png
+  :alt: REV Radio Power Module
+  :width: 500
+
+The `REV Radio Power Module <https://docs.revrobotics.com/rev-11-1856/>`__ is designed to keep one of the most critical system components, the OpenMesh OM5P-AC WiFi radio, powered in the toughest moments of the competition. Traditional barrel jacks easily work themselves loose and often require hacks like hot glue to prevent intermittent power losses. The Radio Power Module eliminates the need for powering the radio through a traditional barrel power jack. Utilizing 18V Passive POE with two socketed RJ45 connectors, the Radio Power Module passes signal between the radio and roboRIO while providing power directly to the radio. After connecting the radio and roboRIO, easily add power to the Radio Power Module by wiring it to the low-current channels on the Power Distribution Hub utilizing the color coded push button WAGO terminals.
```
Comment (sciencewhiz) [inline source/docs/controls-overviews/control-system-hardware.rst:62]: Remove the part number of the radio since there's two legal open mesh PNs.

Comment (sciencewhiz) [reply source/docs/controls-overviews/control-system-hardware.rst:62]: The part about barrel jacks sounds like marketing and not documentation. Recommend removing.

File: source/docs/controls-overviews/control-system-hardware.rst (hunk 2 above)
Comment (Daltz333) [reply source/docs/controls-overviews/control-system-hardware.rst:44]: Stolen from their product page ;)

File: source/docs/controls-overviews/control-system-hardware.rst [hunk 4]
```diff
@@ -25,23 +25,41 @@ NI roboRIO
...
-Voltage Regulator Module
-------------------------
+REV Power Distribution Hub
+--------------------------
+
+.. image:: images/control-system-hardware/power-distribution-hub.png
+  :alt: REV Power Distribution Hub
+  :width: 500
+
+The `REV Power Distribution Hub <https://docs.revrobotics.com/rev-11-1850/>`__ (PDH) is designed to distribute power from a 12VDC battery to various robot components. With 20 high-current (40A max) channels, 3 low-current (15A max), and 1 switchable low-current channel. The Power Distribution Hub features toolless latching WAGO terminals, an LED voltage display, and the ability to connect over CAN or USB-C to the REV Hardware Client for real-time telemetry.
```
Comment (NoahAndrews) [inline source/docs/controls-overviews/control-system-hardware.rst:44]: `With 20 high-current (40A max) channels, 3 low-current (15A max), and 1 switchable low-current channel.` is a sentence fragment.

**Review by NoahAndrews — CHANGES_REQUESTED** [review]

**Review by sciencewhiz — DISMISSED** [review]

File: source/docs/hardware/sensors/serial-buses.rst [hunk 5]
```diff
@@ -106,6 +106,6 @@ Several sensors primarily use the CAN bus. Some examples include:
 - `CAN Based Time-of-Flight Range/Distance Sensor from playingwithfusion.com <https://www.playingwithfusion.com/productview.php?pdid=96&catid=1009>`__
 - TalonSRX-based sensors, such as the `Gadgeteer Pigeon IMU  <http://www.ctr-electronics.com/sensors/gadgeteer-imu-module-pigeon.html>`__ and the `SRX MAG Encoder <https://www.ctr-electronics.com/sensors/srx-magnetic-encoder.html>`__
 - `CANifier <https://www.ctr-electronics.com/control-system/can-can-canifier-driver-led-driver-gpio.html>`__
-- Power monitoring sensors built into the :ref:`Power Distribution Panel (PDP) <docs/controls-overviews/control-system-hardware:Power Distribution Panel>`
+- Power monitoring sensors built into the :ref:`CTRE Power Distribution Panel (PDP) <docs/controls-overviews/control-system-hardware:CTRE Power Distribution Panel>` and the REV Power Distribution Hub (PDH)
```
Comment (jasondaming) [inline source/docs/hardware/sensors/serial-buses.rst:109]:
```suggestion
- Power monitoring sensors built into the :ref:`CTRE Power Distribution Panel (PDP) <docs/controls-overviews/control-system-hardware:CTRE Power Distribution Panel>` and the :ref:`REV Power Distribution Hub (PDH) <docs/controls-overviews/control-system-hardware:REV Power Distribution Hub>`
```

**Review by jasondaming — APPROVED** [review]

**Pull request merged**